optimizely/notification/manager.go

```diff
@@ -40,13 +41,21 @@ func NewAtomicManager() *AtomicManager {
 	}
 }
 
-// AddHandler adds the given handler
-func (am *AtomicManager) AddHandler(newHandler func(interface{})) (int, error) {
+// Add adds the given handler
+func (am *AtomicManager) Add(newHandler func(interface{})) (int, error) {
 	atomic.AddUint32(&am.counter, 1)
 	am.handlers[am.counter] = newHandler
 	return int(am.counter), nil
 }
 
+// Remove removes handler with the given id
+func (am *AtomicManager) Remove(id int) {
+	counter := uint32(id)
```

Comment:
nit: This isn't really a counter. Let's call it `handlerID`

Reply:
fixed.